buildSrc/build.gradle

```diff
@@ -112,6 +112,8 @@ dependencies {
   api 'commons-io:commons-io:2.7'
   api "net.java.dev.jna:jna:5.5.0"
   api 'com.github.jengelman.gradle.plugins:shadow:6.0.0'
+  api 'org.jdom:jdom2:2.0.6.1'
```

Comment:
Can you please also add this changes to the changelog file?

Reply:
I don't think these are used in 1.3 actually, so I'll remove them